src/app/shared/services/dialog/dialog.service.ts

```diff
@@ -105,7 +120,25 @@ export class DialogService {
     return Observable.of(null);
   }
 
-  private alertConfirmDialog(dialogType: DialogType, ...args: Array<any>): Observable<void> {
+  private customSimpleDialog(
+    dialogType: DialogType,
+    config: CustomAlertConfig | CustomConfirmConfig
+  ): Observable<void> {
+
+    const dialogConfig = Object.assign(
+      {
+        component: dialogType === DialogTypes.ALERT ? CustomAlertComponent : CustomConfirmComponent,
```

Comment:
Do we really need 2 different components for this?
Please try to add only one and pass required params inside it